app/frontend/styles/components/step-by-step-navigation.scss

```diff
@@ -0,0 +1,642 @@
+  }
+}
+
+.app-step-nav__step:last-child:before {
+  content: "";
+  position: absolute;
+  z-index: 6;
+  bottom: 0;
+  left: 0;
+  margin-left: 6.5px;
+  width: 13px;
+  height: 0;
+  border-bottom: solid 2px $govuk-border-colour;
+}
+
+.app-step-nav__step:last-child:after {
+  height: -webkit-calc(100% - 15px);
+  height: calc(100% - 15px);
+}
+
+.app-step-nav__step:last-child .app-step-nav__help:after {
+  height: 100%;
+}
+
+@include govuk-media-query($from: tablet) {
+  .app-step-nav--large .app-step-nav__step:last-child:before {
+    margin-left: 8.75px;
+    width: 17.5px;
+    border-width: 3px;
+  }
+
+  .app-step-nav--large .app-step-nav__step:last-child:after {
+    height: calc(100% - 30px);
+  }
+}
+
+.app-step-nav__step--active .app-step-nav__circle--number,
+.app-step-nav__step--active .app-step-nav__help:after,
+.app-step-nav__step--active .app-step-nav__substep:after,
+.app-step-nav__step--active:after,
+.app-step-nav__step--active:last-child:before {
+  border-color: govuk-colour("black");
+}
+
+.app-step-nav__circle {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  position: absolute;
+  z-index: 5;
+  top: 15px;
+  left: 0;
+  width: 26px;
+  height: 26px;
+  color: govuk-colour("black");
+  background: #fff;
+  border-radius: 100px;
+  text-align: center;
+}
+
+@include govuk-media-query($from: tablet) {
+  .app-step-nav--large .app-step-nav__circle {
+    top: 30px;
+    width: 35px;
+    height: 35px;
+  }
+}
+
+.app-step-nav__header:hover .app-step-nav__circle {
+  color: $govuk-link-colour;
+}
+
+.app-step-nav__circle--number {
+  @include govuk-font(16, $weight: bold);
+  line-height: 23px;
+  border: solid 2px $govuk-border-colour;
+}
+
+@include govuk-media-query($from: tablet) {
+  .app-step-nav__circle--number {
+    font-size: 16px;
+    line-height: 23px;
+  }
+}
+
+.app-step-nav--large .app-step-nav__circle--number {
+  @include govuk-font(16, $weight: bold);
+  line-height: 23px;
+}
+
+@include govuk-media-query($from: tablet) {
+  .app-step-nav--large .app-step-nav__circle--number {
+    font-size: 19px;
+    line-height: 30px;
+  }
+}
+
+@include govuk-media-query($from: tablet) {
+  .app-step-nav--large .app-step-nav__circle--number {
+    border-width: 3px;
+  }
+}
+
+.app-step-nav__circle--logic {
+  @include govuk-font(16, $weight: bold);
+  line-height: 28px;
+}
+@include govuk-media-query($from: tablet) {
+  .app-step-nav__circle--logic {
+    font-size: 16px;
+    line-height: 28px;
+  }
+}
+
+.app-step-nav--large .app-step-nav__circle--logic {
+  @include govuk-font(16, $weight: bold);
+  line-height: 28px;
+}
+
+@include govuk-media-query($from: tablet) {
+  .app-step-nav--large .app-step-nav__circle--logic {
+    font-size: 19px;
+    line-height: 34px;
+  }
+}
+
+.app-step-nav__circle-inner {
+  float: right;
+  min-width: 100%;
+}
+
+.app-step-nav__circle-background {
+  text-shadow: 0 -0.1em 0 #fff, 0.1em 0 0 #fff, 0 0.1em 0 #fff, -0.1em 0 0 #fff;
```

Comment:
use of hex values like this should not be allowed in SASS but we have no linting at present. again one for my list/backlog